plugins/importer/src/main/java/com/managemc/importer/reader/model/EssentialsMute.java

```diff
@@ -0,0 +1,40 @@
+package com.managemc.importer.reader.model;
+
+import lombok.Getter;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+public class EssentialsMute {
```

Comment:
POJO (plain old Java object) closely resembling EssentialsX's mute format (documented [here](https://docs.managemc.com/guides/developers/contributing-to-importer/#essentialsx))